cmd/expiration-mailer/main.go

```diff
@@ -229,16 +230,56 @@ func (m *mailer) findExpiringCertificates() error {
 			m.log.Err(fmt.Sprintf("expiration-mailer: Error loading certificates: %s", err))
 			return err // fatal
 		}
-		if len(certs) > 0 {
-			processingStarted := m.clk.Now()
-			m.processCerts(certs)
-			m.stats.TimingDuration("Mailer.Expiration.ProcessingCertificatesLatency", time.Since(processingStarted), 1.0)
+
+		if len(certs) == 0 {
+			continue // nothing to do
 		}
+
+		certs = m.eliminateRenewedCerts(certs)
+
+		processingStarted := m.clk.Now()
+		m.processCerts(certs)
+		m.stats.TimingDuration("Mailer.Expiration.ProcessingCertificatesLatency", time.Since(processingStarted), 1.0)
 	}
 
 	return nil
 }
 
+func (m *mailer) certIsRenewed(serial string) (renewed bool, err error) {
+	present, err := m.dbMap.SelectInt(`
+		SELECT b.serial IS NOT NULL
+		FROM fqdnSets a
+		LEFT OUTER JOIN fqdnSets b
+			ON a.setHash = b.setHash
+			AND a.issued < b.issued
+		WHERE a.serial = :serial
+		LIMIT 1`,
+		map[string]interface{}{"serial": serial},
+	)
+	return present == 1, err
+}
+
+// eliminateRenewedCerts returns a new slice with the already-renewed
+// certificates removed.
+func (m *mailer) eliminateRenewedCerts(certs []core.Certificate) []core.Certificate {
```

Comment:
I don't think we need a separate function to do this. Instead it should probably be merged with [this loop](https://github.com/letsencrypt/boulder/blob/mailer-renewal/cmd/expiration-mailer/main.go#L155) in `processCerts` so we aren't duplicating the work.